AlfrescoApp/App Delegate/AppDelegate.m

```diff
@@ -163,7 +163,15 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
                 [[LoginManager sharedManager] attemptLoginToAccount:accountManager.selectedAccount networkId:accountManager.selectedAccount.selectedNetworkId completionBlock:^(BOOL successful, id<AlfrescoSession> alfrescoSession, NSError *error) {
                     if (!successful)
                     {
-                        displayErrorMessage([ErrorDescriptions descriptionForError:error]);
+                        if (accountManager.selectedAccount.password)
+                        {
+                            displayErrorMessage([ErrorDescriptions descriptionForError:error]);
+                        }
+                        else
+                        {
+                            // Missing password - possibly first launch of an MDM-configured account
+                            displayWarningMessageWithTitle(NSLocalizedString(@"accountdetails.fields.confirmPassword", @"Confirm password"), NSLocalizedString(@"accountdetails.header.authentication", "Account Details"));
```

Comment:
Would be nicer to have a new string here, but I don't want to delay the localisation process